BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_E/Part_E_PUEO.sh

```diff
@@ -24,32 +24,21 @@ IceMCExec=$9
 XFProj=${10}
 PSIMDIR=${11}
 exp=${12}
-#chmod -R 777 /fs/ess/PAS1960/BiconeEvolutionOSC/BiconeEvolution/
 
 
-#cd $WorkingDir
-# put the actual bicone results in the run name directory
-#cp ARA_Bicone_Data/AraOut_Actual_Bicone_Fixed_Polarity_2.9M_NNU.txt Run_Outputs/$RunName/AraOut_ActualBicone.txt
-
-cd $WorkingDir/Antenna_Performance_Metric/
-
-#source set_plotting_env.sh
-
 echo 'Starting fitness function calculating portion...'
 
-mv -f *.root $WorkingDir/Run_Outputs/$RunName/RootFilesGen${gen}/
```

Comment:
Is this just not doing anything right now? Otherwise are we going to be putting the root files in the right location?

Reply:
That line does nothing. I'm not sure if root files were ever even in AntennaPerformanceMetric in the old version. But with this change, I changed the batch jobs to move the simulation executable and output directory into TMPDIR to match what OSC told us to do for Nebulous. Now the custom skimmed root files are moved directly into the file for the specific individual from the outputs in TMPDIR